data/benefits/aide-locale-au-bafa-caf-de-la-savoie.yml

```
@@ -0,0 +1,34 @@
+label: Aide locale au Bafa (CAF de la Savoie)
```

Même commentaire qu'ici #1948 (comment)
```suggestion
label: aide locale au Bafa (CAF de la Savoie)
```

---

`aide au Bafa pour la formation générale (CAF de la Savoie)`